demo/full/scripts/controllers/ContentList.jsx

```diff
@@ -32,35 +35,41 @@ const URL_DENOMINATIONS = {
   DirectFile: "URL to the content",
 };
 
+function formatContentForDisplay(content) {
+  let name = content.name;
+  let disabled = false;
+
+  if (IS_HTTPS) {
+    if (content.url.startsWith("http:")) {
+      name = "[HTTP only] " + name;
+      disabled = true;
+    }
+  } else if (!HAS_EME_APIs &&
+             content.drmInfos &&
+             content.drmInfos.length
+  ) {
+    name = "[HTTPS only] " + name;
+    disabled = true;
+  }
+
+  if (content.live) {
+    name += " (live)";
+  }
+
+  const localContent = !!content.localContent;
+  return { content, name, disabled, localContent };
```

Comment:
What's the difference between `localContent` and `content.localContent`? Why is there both?